admin/schroot/files/schroot.init

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh /etc/rc.common
+
+START=99
+
+start() {
+	[ -d /var/lib/schroot ] || {
```

Comment:
I'm not sure this belongs here. This doesn't start any program or daemon. It just creates directories. This belongs in postinst.

Reply:
side note: /var links to /tmp=tmpfs ...

Reply:
Ah never mind.